authz-model/src/main/resources/canon/authz.json

```diff
@@ -612,6 +626,17 @@
 	            }
           }
         },
+        "PlainText":{
+	        "type": "object",		
+	        "facade":true,
```

Comment:
Unless you need to add hand written code to a class you should set facade to false. Change this and PlainText.java  and IPlainText.java  will disappear, you may have to change a package name where they are used.